server/authn/oidc/oidc.go

```diff
@@ -127,7 +134,7 @@ func (oidc *RemoteOidcAuthenticator) fetchKeys() error {
 func (oidc *RemoteOidcAuthenticator) GetKeys() (*keyfunc.JWKS, error) {
 	jwks, err := keyfunc.Get(oidc.JwksURI, keyfunc.Options{
 		Client:          retryablehttp.New().StandardClient(),
```

Comment:
```suggestion
		Client:          oidc.httpClient,
```

Reply:
Why?

Reply:
Thank you! Updated.

Reply:
@elbuo8 it seems wasteful to create a new client every time?